--- origsrc/mathlib-0.11.1/Makefile	2006-02-22 23:59:31.000000000 -0600
+++ src/mathlib-0.11.1/Makefile	2008-01-10 22:02:56.000000000 -0600
@@ -3,8 +3,11 @@
         mersenne.mli mersenne.ml rand.mli rand.ml vector.mli vector.ml stubs.c
 RESULT=mathlib
 
-export NO_CUSTOM=1
 
+# If on a system without dynamic linking (Such as Mac OS X), comment this out
+export NO_CUSTOM=1
+# and uncomment this
+# STATIC=1
 
 
 ML_SOURCES=$(filter %.ml, $(SOURCES))
@@ -16,7 +19,7 @@
 
 CC=gcc
 RANLIB=ranlib
-CFLAGS=-g -O2 -fPIC -DPIC
+CFLAGS=-O2 -pipe  -fPIC -DPIC
 
 OCAMLDOCFLAGS=-sort
 
